src/components/FormikFileInput/FormikFileInput.tsx

```diff
@@ -0,0 +1,97 @@
+import { pulseAnimation } from "@/styles/animations"
+import { Typography } from "@mui/material"
+import { useField } from "formik"
+import { ReactElement, useState } from "react"
+import { useTranslation } from "react-i18next"
+import styled, { css } from "styled-components"
+
+interface FormikFileInputProps {
+  children: ReactElement
```

Comment:
OPTIONAL You can play with docked comments for every component

Reply:
I'll do it later :)